src/components/Calculator.css

```diff
@@ -0,0 +1,51 @@
+/* stylelint-disable no-missing-end-of-source-newline */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.calculator-cont {
+  display: grid;
+  border: 1px solid black;
```

Comment:
- [ ] It would be better to fix the design in order to achieve the needed design. Here is a quick fix that can help.

- Kindly remove `border: 1px solid black` 
- Add `width: 50em` and `margin: auto` to properly center the app. The screenshot below shows how it looks on my system

![Screenshot from 2022-08-02 14-02-34](https://user-images.githubusercontent.com/46388113/182394064-0494f02f-d1d2-4f63-b880-a20f0cdf1eba.png)

The below screenshot is how the app will look like once you have made the changes
![Screenshot from 2022-08-02 14-06-14](https://user-images.githubusercontent.com/46388113/182394897-0ec6d0ae-2d9e-4020-8fb1-d15f45255947.png)